src/Intl.test.jsx

```diff
@@ -0,0 +1,12 @@
+import { locale } from "./components/Wrapper";
+
+jest.mock("react-intl", () => ({
+  ...jest.requireActual("react-intl"),
+  useIntl: () => ({
+    formatMessage: jest.fn(),
+  }),
+}));
+
+test("the locale is either Spanish or English", () => {
+  expect(locale).toBe("en-US" || "es-US");
+});
```

Comment:
💡 Maybe there's a way to force the locale to be a specific language, then test if the correct translations are being used.

Reply:
Addressed:  4ddba6ca798d6eaad72bb736578a5d7350233668